MCprep_addon/world_tools.py

```diff
@@ -170,8 +170,13 @@ def convert_mtl(filepath):
 	mtl = filepath.rsplit(".", 1)[0] + '.mtl'
 	lines = None
 	copied_file = None
-	with open(mtl, 'r') as mtl_file:
-		lines = mtl_file.readlines()
+
+	try:
+		with open(mtl, 'r') as mtl_file:
+			lines = mtl_file.readlines()
+	except Exception as e:
```

Comment:
Can we do more narrow exception handling? e.g. file not found, and then still have a catch all at the end.